control_plane/roles/control_plane_common/tasks/verify_omnia_params.yml

```diff
@@ -91,6 +91,14 @@
     docker_password: "{{ docker_password }}"
   no_log: True
 
+- name: Validate the domain name
```

Comment:
Does this only validate for a FQDN? What can users do when there is no fully qualified domain name in a lab?  For instance, in our lab it is common practice to use a simplified naming convention such as:

`systemname.local`
or
`systemname.localdomain`

Is it possible to use local domain names and have this test pass?

Reply:
@j0hnL, Freeipa requires the hostname to be a FQDN. So we are validating the same. Also when user provisions the servers using cobbler or idrac (omnia control_plane) we set the hostname as part of omnia. So we are setting the hostname in hostname.domainname format.

Reply:
that's what I thought, thanks for clarifying!